### PR TITLE
Uplink sets

### PR DESCRIPTION
**Pull request by madhav-bharadwaj**
### Description
Support for Oneview UplinkSets

### Issues Resolved
NA

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for go 1.11.x + gofmt checks .
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [X] New endpoints supported are updated in the endpoints-support.md file.
- [X] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: oneview/resource_uplink_set.go [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+		Read:   resourceUplinkSetRead,
+		Update: resourceUplinkSetUpdate,
+		Delete: resourceUplinkSetDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"logical_interconnect_uri": {
```
Comment (AnushaY1916) [inline oneview/resource_uplink_set.go]: This is a required field. Please check the documentation once and mark the fields as required which are mentioned as mandatory fields. I see few mandatory fields mentioned as optional.

File: oneview/resource_uplink_set.go [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+				Computed: true,
+			},
+			"eTag": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```
[the remark this answers is not in the document]
Comment (madhav-bharadwaj) [reply oneview/resource_uplink_set.go]: Is this in data source files or resource files as well?

File: oneview/resource_uplink_set.go (hunk 1 above)
Comment (madhav-bharadwaj) [reply oneview/resource_uplink_set.go]: Sorry missed it. Have added all the required tags.

File: oneview/resource_uplink_set.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (AnushaY1916) [reply oneview/resource_uplink_set.go]: Both.

**Comment by madhav-bharadwaj** [comment]
> 
> 
> This resource is missing tests

@mbfrahry , have added the tests for the resouce. Please have a look.

**Pull request merged**